src/ripple/rpc/handlers/LedgerEntry.cpp

```diff
@@ -173,7 +190,12 @@ doLedgerEntry(RPC::JsonContext& context)
         expectedType = ltESCROW;
         if (!context.params[jss::escrow].isObject())
         {
-            uNodeIndex.SetHex(context.params[jss::escrow].asString());
+            // CHECKME We should return an error here
```

Comment:
Yes, I believe we should return an error here. There were a couple other places where you made similar changes and I agree those are correct as well.

Reply:
We do, so the comment is obsolete.